net-wireless/unifi/unifi-5.6.37.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit systemd user
+
+DESCRIPTION="A Management Controller for Ubiquiti Networks UniFi APs"
+HOMEPAGE="https://www.ubnt.com"
+SRC_URI="https://dl.ubnt.com/unifi/${PV}/UniFi.unix.zip -> ${P}.zip"
+
+KEYWORDS="-* ~amd64 ~x86"
+LICENSE="ubiquiti"
+SLOT="5/6"
+
+RDEPEND="dev-db/mongodb
+	dev-java/tomcat-native
+	virtual/jre:1.8"
+
+DEPEND="app-arch/unzip"
+
+RESTRICT="bindist mirror strip"
+
+S="${WORKDIR}/UniFi"
+
+DOCS=( "readme.txt" )
+
+QA_PREBUILT="usr/lib/unifi/lib/native/Linux/x86_64/*.so
```

Comment:
Do they install a directory called `x86_64` on 32-bit systems?

Reply:
Yes. There is no `i386`. I just ran a test on Ubuntu 16.04 32-Bit. It also there install this folder.